Makefile

```diff
@@ -54,11 +54,6 @@ ccoverage:  ## Run the test suite, with C++ code coverage
 	$(MAKE) clean
 	CFLAGS="$(CFLAGS) -O0 -pg --coverage" $(MAKE) build
 	$(MAKE) check
-	gcov -i build/*/src/pystack/_pystack -i -d
```

Comment:
Why are we removing this? This is still useful to run local runs of coverage, no?